periodic_job.go

```diff
@@ -0,0 +1,189 @@
+package river
```

Comment:
Renamed this from `periodic.go` to `periodic_job.go` to be a bit more descriptive.